pkg/api/scylla/v1/cluster_types.go

```diff
@@ -95,6 +95,44 @@ type ScyllaClusterSpec struct {
 	// used for pulling Scylla and Agent images.
 	// +optional
 	ImagePullSecrets []corev1.LocalObjectReference `json:"imagePullSecrets,omitempty"`
+
+	// ingressOptions is an optional Ingress configuration options.
+	// If provided, Ingress objects are generated for each Scylla node with following options.
+	// Experimental - subject to incompatible changes.
+	// +optional
+	IngressOptions *IngressOptions `json:"ingressOptions,omitempty"`
+}
+
+// IngressOptions defines configuration options for Ingress objects associated with cluster nodes.
+// Experimental - subject to incompatible changes.
+type IngressOptions struct {
+	// domain specifies DNS suffix of hostname of each Scylla node.
+	// It is concatenated with node host_id to form Ingress rule pointing to particular Scylla node.
+	Domain string `json:"domain"`
```

Comment:
domain isn't specific to ingresses (say it's also needed for standalone certs) and it also needs to be an array